apps/spruce/src/components/Accordion/index.tsx

```diff
@@ -13,10 +13,13 @@ interface AccordionProps {
   showCaret?: boolean;
   title: React.ReactNode;
   titleTag?: React.FC;
+  /** `toggledTitle` replaces the title element when the accordion is open */
   toggledTitle?: React.ReactNode;
   toggleFromBottom?: boolean;
   useIndent?: boolean;
   subtitle?: React.ReactNode;
+  /** This prop prevents the accordion from rendering the child component if the accordion is collapsed. It is useful if the child is expensive to render. */
+  shouldRenderChildIfHidden?: boolean;
```

Comment:
I wouldn't want to do it in this pr since that also has the consequence of unmounting the component so it could result in unexpected behaviors for other components that use the Accordion component.